packages/compass-saved-aggregations-queries/src/stores/open-item.ts

```diff
@@ -426,15 +426,18 @@ const openItem =
     database: string,
     collection: string
   ): SavedQueryAggregationThunkAction<void> =>
-  (_dispatch, _getState, { track, workspaces }) => {
+  (_dispatch, _getState, { track, workspaces, connectionsManager }) => {
+    const connectionInfo =
+      connectionsManager.getConnectionById(connection)?.info;
```

Comment:
@gribnoysup is this to be the new way of retrieving connection info, or should we stick to the connectionRepository for now so that it's consistent with the rest?

Reply:
It will not be the new way, we will have a single `connections` locator. We will continue removing references to `connectionsManager` and `connectionRepository` from the code, both abstractions don't exist anymore, all the hooks for them are already marked as deprecated, so it doesn't really make a difference right now, connectionsManager is just already available in this partcular store and doesn't require another dependency injection to be added